fluentd/templates/daemonset.yaml

```diff
@@ -0,0 +1,152 @@
+{{- $root := . -}}
+{{- if and .Values.daemonset.enabled (not .Values.statefulset.enabled) }}
```

Comment:
Is there use case `daemonset` and `statefulset` together? For example, you could collect logs in this chart `daemonset` and then forward it to this chart `statefulset`. In that case, is it different for releases?

Reply:
I think daemonset and statefulset should be not managed in a single helm chart.
This is because statefulset has to be `helm uninstalled` once for some changes.